cookie-banner-rules-list.json

```diff
@@ -636,6 +636,265 @@
+  },
+    {
+  "click": {
+    "optIn": "#cmpbntyestxt",
+    "optOut": "#cmpbntnotxt",
+    "presence": "div.cmpboxbtns"
+  },
+  "domain": "sourceforge.net",
+  "schema": 1665669888782,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "__cmpconsentx11319",
+        "value": "CPgx6wgPgx6wgAfUnBENCkCsAP_AAH_AACiQGgwDwAGgAVABAADQAJgAXwAwgB4AEEAJAAUYBCAFoALzAaCBoMA8ABoAFQAQAA0ACYAF8AMIAeABBACQAFGAQgBaAC8wGggAAA",
+        "isSecure": true
+      }
+    ],
+    "optOut": [
+      {
+        "name": "__cmpconsentx11319",
+        "value": "CPgx6wgPgx6wgAfUnBENCkCgAAAAAAAAACiQAAAAAAAA",
+        "isSecure": true
+      }
+    ]
+  },
+  "id": "786f7ae5-548a-43d4-97a2-7b8efb5ed7a7"
+  },
+    {
+  "click": {
+    "optIn": "button#onetrust-accept-btn-handler",
+    "presence": "div#onetrust-button-group-parent"
+  },
+  "domain": "digicert.com",
+  "schema": 1665670173294,
+  "cookies": {},
+  "id": "0da6a0df-beac-4499-b3ac-023fc68ca695"
+  },
+    {
+  "click": {
+    "optIn": "button#onetrust-accept-btn-handler",
+    "optOut": "button#onetrust-reject-all-handler",
+    "presence": "div#onetrust-button-group"
+  },
+  "domain": "webex.com",
+  "schema": 1665670505287,
+  "cookies": {},
+  "id": "5e777243-0819-48a6-90b4-66a1353e2514"
+  },
+    {
+  "click": {
+    "optIn": "div#gdpr-single-choice-overlay",
+    "presence": "button.wt-btn--filled"
+  },
+  "domain": "etsy.com",
+  "schema": 1665671188137,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "p",
+        "value": "eyJnZHByX3RwIjoxLCJnZHByX3AiOjF9"
+      }
+    ]
+  },
+  "id": "e5706488-f2e9-47bd-a178-4d569ca26ce8"
+  },
+    {
+  "click": {
+    "optIn": "button#onetrust-accept-btn-handler",
+    "optOut": "button#onetrust-reject-all-handler",
+    "presence": "div#onetrust-button-group"
+  },
+  "domain": "indeed.com",
+  "schema": 1665671330451,
+  "cookies": {},
+  "id": "a4d94551-fc83-4477-b542-a13950b97468"
+  },
+    {
+  "click": {
+    "optIn": "button#L2AGLb",
+    "optOut": "button#W0wltc",
+    "presence": "div.spoKVd"
+  },
+  "domain": "google.co.in",
+  "schema": 1665671846324,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "NID",
+        "value": "511=LsN6Ln7-Mo3s1--Jj6XriDXGASnZN_hteNJEsV6PLicq77EDsKmJ8VTJZTDocymcQypv1LdFO5ybVOkZEZrNzxAn3T1F1OwY7lqNpB1LRkGd7LKLv_958U0Shm4Q2Pf4eqmSSZt54JGHrkp2bsttfFriwXdN1LkZzPG8DBxcbz0",
+      }
+    ],
+    "optOut": [
+      {
+        "name": "SOCS",
+        "value": "CAESHAgBEhJnd3NfMjAyMjA5MjEtMF9SQzQaAnJvIAEaBgiA49iZBg",
+      }
+    ]
+  },
+  "id": "9c252bbd-0ddc-4677-bab1-3946161dc819"
+  },
+  {
+  "click": {
+    "optIn": "button#onetrust-accept-btn-handler",
+    "presence": "div#onetrust-consent-sdk"
+  },
+  "domain": "weebly.com",
+  "schema": 1665747960527,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "cookie-consent",
+        "value": "{\"allowStrictlyNecessaryCookies\":true,\"allowFunctionalityCookies\":true,\"allowPerformanceCookies\":true,\"allowTargetingCookies\":true}"
```

Comment:
I'm curious whether we could add an opt-out variant of this with these fields set to `false` (except strictly necessary perhaps). Did you try that?

Reply:
From waht I can see the variant with everything set to false except the strictly necessary one is there by default before interacting with the banner and a second cookie UUID cookie is added after customizing the cookie preferences.

Reply:
Ok in that case let's stick to the opt-in rule for now.